### DISCUSSION TITLE
Neon OS Wish List

### DISCUSSION THREAD

**Discussion by NeonDaniel** [NeonGeckoCom/NeonCore#346] [Ideas]
Feature requests or suggested changes go here to be discussed so we can spec out a solution.

**Comment by NeonDaniel** [comment]
Home Assistant support

**Reply by NeonDaniel** [reply]
This is in-progress; an upcoming release will include the HomeAssistant plugin, and there is a community-member effort to get a skill together that interfaces with the plugin

**Comment by NeonDaniel** [comment]
Numeric confirmations for power off/reboot

**Reply by NeonDaniel** [reply]
The rationale for this is that there are some commands we don't want a user to accidentally confirm, so we use a random 3 digit confirmation for a user to speak. I think an on-screen confirmation option would provide a good alternative for devices with GUI input, but open to suggestions for how to make the voice UI better.

**Reply by NeonClary** [reply]
I think if it just read the numbers as single sequential digits it would work better.  "three, four, five" rather than "three hundred and forty-five."  There is quite a bit of variation in how people speak larger numbers. I find it a bit challenging to remember exactly how the computer phrased the number. I do love that it asks for confirmation. Two numbers might be sufficient, or a single distinctive word which could even be the same each time. Aardvark, for instance. Or a friendly phrase confirmation such as "Good-bye Neon"

**Reply by NeonDaniel** [reply]
It should actually work with either pronunciation, one issue is words like "two" and "four" will almost always become "to" and "for", especially since there's no context for STT.

If we don't care to have the extra validation, it could just be "yes" or "no" to continue, or some static phrase as you suggested.

**Comment by NeonDaniel** [comment]
Spoken confirmation while searching for media

**Reply by NeonDaniel** [reply]
Current behavior is a direct implementation of [OCP](https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin). There should be an option added to speak something on intent match.

**Comment by NeonDaniel** [comment]
Alarm skill - Configurable GUI behavior

Provide some setting to leave timers on-screen indefinitely
Make the timeout for alarm card view on set/query configurable

**Comment by NeonDaniel** [comment]
Sleep Timer

Add an option to play media for some duration as a sleep timer, etc.

**Comment by NeonDaniel** [comment]
Add an option to adjust screen brightness by voice
